uid2_client/uid2_token_generator.py

```diff
@@ -124,13 +127,13 @@ def generate_uid2_token_with_debug_info(id_str, master_key, site_id, site_key, p
 
         # User Identity Data
         site_payload += int.to_bytes(0, length=4, byteorder='big')  # privacy bits
-        generated_at_timestamp = int(params.token_generated_at.timestamp()) * 1000
-        site_payload += int.to_bytes(generated_at_timestamp, length=8, byteorder='big')  # established
+        site_payload += int.to_bytes(int(params.identity_established.timestamp()) * 1000, length=8, byteorder='big')  # established
+        generated_at_timestamp = int(params.token_generated.timestamp()) * 1000
         site_payload += int.to_bytes(generated_at_timestamp, length=8, byteorder='big')  # last refreshed/generated
         site_payload += base64.b64decode(id_str)
 
         master_payload = int.to_bytes(int(params.token_expiry.timestamp()) * 1000, length=8, byteorder='big')  # expiry
-        master_payload += int.to_bytes(generated_at_timestamp, length=8, byteorder='big')  # created
+        master_payload += int.to_bytes(generated_at_timestamp, length=8, byteorder='big')  # generated
```

Comment:
Nitpick, do you want to write the same comment as C# SDK [here](https://github.com/IABTechLab/uid2-client-net/blob/main/src/UID2.Client/Utils/UID2TokenGenerator.cs#L134), `//identity refreshed, seems to be identical to TokenGenerated in Operator` to provide more context.

Reply:
I think its understood from the `# generated` comment that its identical to last refreshed/generated in site payload